hoppfish/hwangsae-recorder-user-command.py

```diff
@@ -98,7 +98,8 @@ def get(self, method):
                     }
                     file_list_ret.append(file_data)
                 ret['fileList'] = file_list_ret
-                ret = json.dumps(ret)
+                ret_str = json.dumps(ret)
```

Comment:
I think it's better to use `flask.json.jsonify()` instead of introducing a new temporary variable.